tests/attacks/test_carlini.py

```diff
@@ -48,10 +48,10 @@ def setUpClass(cls):
 
         cls.n_train = 10
         cls.n_test = 10
-        cls.x_train_mnist = cls.x_train_mnist[0 : cls.n_train]
-        cls.y_train_mnist = cls.y_train_mnist[0 : cls.n_train]
-        cls.x_test_mnist = cls.x_test_mnist[0 : cls.n_test]
-        cls.y_test_mnist = cls.y_test_mnist[0 : cls.n_test]
+        cls.x_train_mnist = cls.x_train_mnist[0: cls.n_train]
+        cls.y_train_mnist = cls.y_train_mnist[0: cls.n_train]
+        cls.x_test_mnist = cls.x_test_mnist[0: cls.n_test]
+        cls.y_test_mnist = cls.y_test_mnist[0: cls.n_test]
```

Comment:
Please revert these changes because they are not following Black style formatting.
```suggestion
        cls.x_train_mnist = cls.x_train_mnist[0 : cls.n_train]
        cls.y_train_mnist = cls.y_train_mnist[0 : cls.n_train]
        cls.x_test_mnist = cls.x_test_mnist[0 : cls.n_test]
        cls.y_test_mnist = cls.y_test_mnist[0 : cls.n_test]
```

Reply:
My mistake, I thought Black style is PEP8 compliance, so when the IDE issued a warning of PEP8 infriginment, I did change them as suggestions. Now it recovered.